MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include bioio *.toml
```

Comment:
If anyone knows how to do this in pyproject.toml that is probably better.

Reply:
is this trying to include the dummy plugin?  I think for the manifest we want to exclude test files right?  or am i misunderstanding

Reply:
also there's a check-manifest section in the main pyproject.toml